src/components/ConnectWallet.vue

```diff
@@ -252,9 +252,15 @@
 </template>
 
 <script>
+import metamask from "@/assets/icons/metamask.svg"
```

Comment:
be more specific on these, i would have assumed thats a whole component
```suggestion
import metamaskSvg from "@/assets/icons/metamask.svg"
```